src/llmq/signing.cpp

```diff
@@ -574,7 +582,7 @@ bool CSigningManager::ProcessPendingRecoveredSigs(PeerManager& peerman)
         }
     }
 
-    return recSigsByNode.size() >= nMaxBatchSize;
```

Comment:
@coderabbitai review if the old / replaced logic was actually buggy, or if it was acceptable and as intended.